README.md

```diff
@@ -142,6 +142,8 @@ This project wouldn't be possible without our awesome contributors. Thank you >:
             <a href="https://www.facebook.com/melnic.ira/"><img src="https://raw.githubusercontent.com/FrontEnd-ro/frontend.ro/master/client/public/images/contributors/iramelnic.jpg" width="120px;" alt="Ira Melnic"/><br /><sub><b>Ira Melnic</b></sub></a></td>
         <td align="center">
             <a href="https://lazarnarcis.github.io/"><img src="https://raw.githubusercontent.com/FrontEnd-ro/frontend.ro/master/client/public/images/contributors/lazarnarcis.png" width="120px;" alt="Narcis Lazar"/><br /><sub><b>Narcis Lazar</b></sub></a></td>
+        <td align="center">
+            <a href="https://sergiu-szatmari.github.io/"><img src="https://raw.githubusercontent.com/FrontEnd-ro/frontend.ro/master/client/public/images/contributors/sergiu-sz.png" width="120px;" alt="Sergiu Szatmari"/><br /><sub><b>Sergiu Szatmari</b></sub></a></td>
```

Comment:
A couple of small things:

1. As mentioned in [CONTRIBUTING.md](https://github.com/FrontEnd-ro/frontend.ro/blob/master/CONTRIBUTING.md#commit-messages) - every commit must have an issue associated. So please create an issue in the repo regarding this task
2. Please rename the photo according to your full github username. This is the pattern we're using
3. Squash all commits into one. I will merge after that :D